lib/ledger.js

```diff
@@ -399,17 +408,17 @@ async function _sendElectorPool({electorPoolId, maintainerDoc}) {
     capabilityAction: 'write',
     proofValue: 'MOCKPROOF',
     proofPurpose: 'capabilityInvocation',
+    invocationTarget: maintainerDoc.didDocument.id,
```

Comment:
Don't do this. Pass in the ledger ID. If you do this, when someone fixes/changes the ledger ID to be something else, they will miss this and we'll have a bug on our hands.

Reply:
good idea and done here: https://github.com/veres-one/veres-one/pull/44/commits/113e33310996b7596530ea90364b74a1abacce90